Use constants for Vert.x web route order marks (#35917)
quarkusio/quarkus: docs/src/main/asciidoc/reactive-routes.adoc

}
----

> === Well known route order values

Reviewer: Hm, it's a `vertx-http` thing but it's documented in the reactive routes reference guide. I wonder if we should place this information in the [HTTP Reference ](https://quarkus.io/guides/http-reference) and add a link here instead?

Reviewer 2: Yes, wrong place, it should be in the HTTP reference.

Author: Moved to the HTTP reference, added a new chapter at the end.

> === Well known route order values

Reviewer 2: ```suggestion
=== Well-known route order values
```

> Quarkus uses a couple of route order marks itself. Your route order mark values should be at least positive integer

Reviewer 2: ```suggestion
Quarkus uses a couple of route order marks itself. Your route order mark values should be at least a positive integer
```

> values above `20000` (see `RouteConstants.ROUTE_ORDER_AFTER_DEFAULT_MARK`) to not interfere with functionality provided

Reviewer 2: ```suggestion
value above `20000` (see `RouteConstants.ROUTE_ORDER_AFTER_DEFAULT_MARK`) to not interfere with the functionality provided
```
